components/Banner/src/MDCBannerView.m

```diff
@@ -375,13 +375,23 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return CGSizeMake(size.width, frameHeight);
 }
 
+- (CGSize)intrinsicContentSize {
+  CGFloat intrinsicContentHeight = [self sizeThatFits:self.bounds.size].height;
```

Comment:
Is it safe to rely on self.bounds.size here? This logic reads to me as self-referential.

Reply:
I think it is safe to rely on the `self.bounds.size` here. The reason behind it is only `width` from `self.bounds` is used, and `intrinsicContentSize` doesn't add additional change/constraints to width. This pattern is also used in some other components. Let's follow up offline on it.

Reply:
If you keep it, please use `CGRectGetHeight(self.bounds)`.